Simple {{fa-icon "coffee"}} not working in IE11 because of Object.assign in didReceiveAttrs

Trying to render a simple `{{fa-icon "coffee"}}` on IE11 gives an error:

> Object doesn't support property or method 'assign'

The problem is the following line in the `didReceiveAttrs` function of `addon/components/fa-icon.js`:

```
const o = Object.assign({},
      classes,
      transform,
      mask,
      {symbol: symbol}
    )
```

[IE11 doesn't support Object.assign](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign). 

Adding IE11 to my targets.js file also doesn't help here.

From what I can tell, FA5 [supports IE11](https://fontawesome.com/how-to-use/on-the-web/other-topics/browser-support).

Ember also [officially supports IE11](https://github.com/kaliber5/ember-bootstrap/issues/454) unless I'm mistaken.

I couldn't see any comments about limited browser support for this add-on. Is this hence a bug or is IE11 not supported by this add-on?

So I'm a bit confused about how to make this contribution. I tried to follow the contribution guidelines but got stuck at the `npm publish` stage because of a:

```
Failed PUT 404
npm ERR! code E404
npm ERR! 404 Not found : @fortawesome/ember-fontawesome
npm ERR! 404
npm ERR! 404  '@fortawesome/ember-fontawesome' is not in the npm registry.
```

So I created a PR from a forked repository instead. Was this a mistake, and if so how should I make this contribution?

--- a/addon/components/fa-icon.js
+++ b/addon/components/fa-icon.js
@@ -4,6 +4,7 @@ import Ember from 'ember'
 import { icon, parse, toHtml, config } from '@fortawesome/fontawesome-svg-core'
 import { htmlSafe } from '@ember/string'
 import { computed, getWithDefault } from '@ember/object'
+import { assign } from '@ember/polyfills';
 import appConfig from 'ember-get-config';
 
 function getConfigOption (key, defaultValue) {
@@ -101,7 +102,7 @@ const IconComponent = Component.extend({
     const mask = objectWithKey('mask', normalizeIconArgs(null, this.get('mask')))
     const symbol = this.getWithDefault('symbol', false)
 
-    const o = Object.assign({},
+    const o = assign({},
       classes,
       transform,
       mask,

